src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -49,6 +49,8 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
             return text[characterPosition] == '.';
         }
 
+        internal override ImmutableHashSet<char> PossibleTriggerCharacters => ImmutableHashSet.Create('.');
```

Comment:
can you make these all `{ get; } =`.  there's no point allocating this garbage over and over again.